### PR TITLE
wsusserver: support product families

### PR DESCRIPTION
**Pull request by nathangiuliani**
#### Pull Request (PR) description
Added the ability to specify product families as well as products.
Added the ability to specify all products.
Modified product/product family PowerShell code to state what it is adding/removing.

#### This Pull Request (PR) fixes the following issues
n/a

#### Task list
- [x] Resource/Class documentation added/updated in README.md?
- [x] Examples appropriately added/updated?
- [x] Unit tests added/updated?
- [ ] Integration tests added/updated (where possible)?

### CODE REVIEW

**Comment by TraGicCode** [comment]
Hey @nathangiuliani ,

Can you rebase with master?

**Comment by nathangiuliani** [comment]
> Hey @nathangiuliani ,
> 
> Can you rebase with master?

done

File: README.md [hunk 1]
```diff
@@ -189,6 +195,10 @@ class { 'wsusserver':
       'SQL Server Feature Pack',
       'SQL Server 2012 Product Updates for Setup',
       'Windows Server 2016',
+    ],
+        product_families                   => [
```
Comment (TraGicCode) [inline README.md]: can you fix the spacing/tabing here?

**Review by TraGicCode — CHANGES_REQUESTED** [review]
Minor formatting change

**Comment by TraGicCode** [comment]
Hey @nathangiuliani ,

Thanks for working on this!

**Review by TraGicCode — APPROVED** [review]
LGTM

**Pull request merged**